nuboxdevcom/maintenance-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "NDC\\MaintenanceBundle\\NDCMaintenanceBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "composer-scripts": {
```

Comment:
You do not need to include the composer-scripts. They are already included by other bundles (framework bundle)